### PR TITLE
ATS-468 : Add Veracode (SAST & SCA) scans to Transform Service Travis builds

### PR DESCRIPTION
**Pull request by DenisGabriela**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -43,6 +43,14 @@ jobs:
       before_install: bash _ci/init.sh
       script: bash _ci/whitesource.sh
 
+    - name: "Source Clear Scan"
+      stage: build
+      if: branch NOT IN (company_release)
+      before_install: bash _ci/init.sh
+      script: skip
```
Comment (montgolfiere) [inline .travis.yml]: Can we document somewhere (JIRA, Confluence, README ?) how this is configured ?  For example, policies that will cause the build to fail ?

Comment (DenisGabriela) [reply .travis.yml]: I've added a few details here. Thanks for review.

**Comment by tzclucian** [comment]
> We should not delete WhiteSource yet - not until we're fully up-to-speed with VeraCode.

It was deleted upon [this chat](https://alfresco.slack.com/archives/CT2NQUYE4/p1584438966008200?thread_ts=1584374888.003800&cid=CT2NQUYE4) on the #eng-security-veracode channel.
As I'm seing two sides on this topic ... which one should we follow? Is there any decision/documentation regarding this at an organizational level that we should stick to?.

cc: @mteodori

**Comment by DenisGabriela** [comment]
Can I have one more approve here as well? Similar changes on transform-service have been approved&merged.

**Pull request merged**